src/mainboard/raptor-cs/talos-2/devicetree.dts

```diff
@@ -41,20 +41,41 @@
 				ibm,chip-id = <0x00>;
 			};
 
+			sensor@4 {
+				compatible = "ibm,ipmi-sensor";
+				ibm,chip-id = <0x08>;
+				ipmi-sensor-type = <0x07>;
```

Comment:
Ok, there are two such entries per CPU because one of them is for OCC state, according to talos.xml:

```xml
<globalSetting>
	<id>/sys-0/node-0/motherboard-0/proc_socket-1/module-0/p9_proc_s/occ/occ_active_sensor</id>
	<property>
	<id>IPMI_SENSOR_ID</id>
	<value>0x04</value>
	</property>
</globalSetting>
```

Reply:
Right, it's probably that sensor which enables communication of BMC with OCC.